modules/analysis-common/src/test/resources/rest-api-spec/test/analysis-common/30_tokenizers.yml

```diff
@@ -27,6 +27,23 @@
     - match:  { detail.tokenizer.tokens.2.token: od }
 
 ---
+"nGram_exception":
+    - skip:
+        version: " - 6.0.0"
```

Comment:
@jimczi  Jim,  I am not sure about this `6.0.0` number.  I assumed since we are on `6.0.1` now, I would skip up to `6.0.0`.

Or should this rather be `6.0.99`?

Reply:
This pr targets 6.x, that's 6.1.0. We don't want to break the build so even if you plan to backport to 6.0.1 you need to do it in multiple steps. The first commit sets the expectation to the current version (so 6.0.99 here), otherwise the bwc tests would fail and you can change the expectations only after the backport to previous versions.
For this deprecation, I think it's ok to not backport to 6.0.1 and start only in 6.1.0.